tex/gregoriotex-syllable.tex

```diff
@@ -213,6 +214,9 @@
   \or %
     % case of divisiofinalis
     \global\setbox\gre@box@temp@width=\hbox{\gre@fontchar@divisiofinalis}%
+  \or %
+    % case of no note
+    \global\setbox\gre@box@temp@width=\hbox{}%
```

Comment:
You know, it's occured to me that given that no note syllables have their own type here, it would be possible to modify the implementation to leave the output of the old bar spacing algorithm unchanged.  I realize that we don't want to delay rc1, but is this something I should do for rc2?